lib/clone.js

```diff
@@ -0,0 +1,22 @@
+const fs = require('fs')
+const CSV = require('csv-string')
+const simpleGit = require('simple-git')
+
+const clone = () => {
+  fs.readFile('status.csv', 'utf8', function (err,data) {
```

Comment:
reading the status.csv seems repetitive.

Personally i'd suggest the code was better architected as a simple class which takes existing dictionary of info and returns a dictionary and then a wrapper loads and writes from disk e.g.

```
class CoreTools {
  constructor(statusInfoAsArrayOfObjectsOrCsvFilePath) {
  }

  checkAll() {
  }
  
  etc
}


```